Content.Server/Actions/ActionOnInteractSystem.cs

```diff
@@ -72,16 +79,16 @@ private void OnAfterInteract(EntityUid uid, ActionOnInteractComponent component,
         if (args.Handled)
             return;
 
-        if (component.ActionEntities is not {} actionEnts)
+        if (component.ActionEntities is not { } actionEnts)
         {
-            if (!TryComp<ActionsContainerComponent>(uid,  out var actionsContainerComponent))
+            if (!TryComp<ActionsContainerComponent>(uid, out var actionsContainerComponent))
                 return;
 
             actionEnts = actionsContainerComponent.Container.ContainedEntities.ToList();
         }
 
         // First, try entity target actions
-        if (args.Target is {} target)
+        if (args.Target is { } target)
```

Comment:
You didn't add this, but I saw it come up on discord and this this bit of code here and in the other method should be changed, so you can probably also include that in this PR.

If the component should pick randomly from all available actions, then it should just always get the actions from `actionsContainerComponent`, and not be performing a one-off clone of the list of actions whenever it is first used.

If instead the component should only pick from the action prototypes that it itself added via `ActionOnInteractComponent.Actions`, then it should just populate `ActionOnInteractComponent.ActionEntities` when the entities are spawned on map init, not here.